quickstarts/WFLY-16195-upgrade-security-quickstarts.adoc

```diff
@@ -0,0 +1,160 @@
+
+* ejb-security-programmatic-auth
+** `configure-elytron.cli` has one redundant command that sets the `sasl-authentication-factory` in the `http-connector`
+even though this is no longer necessary.
+** `configure-elytron.cli` also adds an `application-security-domain` mapping for "quickstart-domain" in the EJB subsystem.
+This extra configuration isn’t necessary anymore since there's a default `application-security-domain` mapping for "other"
+in the EJB subsystem. That could be used directly instead.
+
+* ejb-security
+** The entire configure-elytron.cli script is now redundant. This quickstart will be removed via https://issues.redhat.com/browse/WFLY-16140[WFLY-16140].
+
+** A build issue needs to be fixed ("`dependencies.dependency.version` for `org.jboss.resteasy:resteasy-jaxrs:jar` is missing").
+
+** Deploying the application fails with the following error because the EJB and Undertow subsystems are mapping "other" to
+different Elytron security domains. This needs to be fixed.
+
+The following quickstarts require updates to their READMEs:
+
+* servlet-security
+** The "Review the Modified Server Configuration" section mentions legacy security. Those references need to be removed.
+** The "Server Log: Expected Warnings and Errors" section indicates that a warning will appear in the server log.
+This warning no longer appears so this can now be removed.
+
+* microprofile-jwt
+** In the "Starting from Scratch" section, `<version>${version.wildfly.maven.plugin}</version>` needs to be removed
+from the step that adds the `wildfly-maven-plugin` to the `pom.xml` in order to get the build to succeed.
+** In the "Activating MicroProfile JWT" section, it says to add the `LoginConfig` annotation. However, this is already
+added when creating `App.java` in the "JAX-RS Conversion" section so this step should be removed.
+
+* security-domain-to-domin
+** The README indicates that a warning will appear in the server log. This warning no longer appears so this section
+should be removed.
+
+=== Nice-to-Have Requirements
+
+=== Non-Requirements
+
+== Backwards Compatibility
+
+// Does this enhancement affect backwards compatibility with previously released
+// versions of WildFly?
+// Can the identified incompatibility be avoided?
+
+=== Default Configuration
+
+=== Importing Existing Configuration
+
+=== Deployments
+
+=== Interoperability
+
+//== Implementation Plan
+////
+Delete if not needed. The intent is if you have a complex feature which can 
+not be delivered all in one go to suggest the strategy. If your feature falls 
+into this category, please mention the Release Coordinators on the pull 
+request so they are aware.
+////
+
+== Security Considerations
+
+////
+Identification if any security implications that may need to be considered with this feature
+or a confirmation that there are no security implications to consider.
+////
+
+== Test Plan
```

Comment:
What is the expected scope of testing on the engineering side? Just manual verification? Do the QSs have some (unit) tests (within the project) that can be leveraged? Do you plan to add some tests?

Reply:
On the engineering side, just manual verification will be done by following the steps in the quickstart READMEs. I've updated the analysis to reflect this.